src/renderer/src/routes/main/apps-view/+page.svelte

```diff
@@ -22,15 +52,20 @@
 	<div class="relative mx-2 flex-grow">
 		<Input
 			bind:value={searchInput}
+			bind:autocomplete
+			on:keydown={handleEnterPress}
 			props={{
 				class: 'pl-10 input rounded placeholder-tertiary-500 text-base font-medium',
 				type: 'text',
-				placeholder: $i18n.t('whatDoYouWantToLaunch')
+				placeholder: $i18n.t('whatDoYouWantToLaunch'),
+				autofocus: true
```

Comment:
This does not seem to work for me. Maybe we need to call focus() on the search input explicitly whenever the window itself receives focus.